server/models/express-usage-record.js

```diff
@@ -35,4 +54,84 @@ module.exports = function extendExpressUsageRecord(ExpressUsageRecord) {
     });
   }
   ExpressUsageRecord.recordUsage = recordUsage;
+
+  function dailySummary(callback) {
+    ExpressUsageRecord.find({order: ['lbModelName', 'requestUrl']},
+      function(err, records) {
+        if (err) return callback(err);
+        var bucket = {};
+        for (var r in records) {
+          if (!records.hasOwnProperty(r)) continue;
+          var key = records[r].lbModelName;
+          if (!key) {
+            key = records[r].requestUrl;
+          }
+
+          if (!bucket.hasOwnProperty(key)) bucket[key] = 0;
+          bucket[key] += 1;
+        }
+        callback(null, bucket);
+      }
+    );
+  }
+  ExpressUsageRecord.dailySummary = dailySummary;
+
+  function hourlySummary(modelOrUri, window, callback) {
+    var windowStart = new Date(window);
```

Comment:
Uff, `window` is a global variable in the browser, it took me a while to realise that `new Date(window)` is referring to a function argument instead. Would you mind using a different name, e.g. `windowStartTime `?